src/domains/shopping-cart/hooks/useSelectedCartIds.ts

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+
+import { getLocalStorage, setLocalStorage } from "../../../utils/localStorage";
+import { CartItemTypes } from "../types/cartItem";
+
+export function useSelectedCartIds() {
+  const [selectedCartIds, setSelectedCartIds] = useState<string[]>(
+    getLocalStorage("selectedCartIds", [])
+  );
```

Comment:
```suggestion
  const [selectedCartIds, setSelectedCartIds] = useState<string[]>(
    () => getLocalStorage("selectedCartIds", [])
  );
```

캉골이 사용한 방식과 이렇게 Lazy initialization 방식 사이에 무슨 차이점이 있나요?
해당 방식을 사용한 이유가 있다면 궁금합니다!